src/App.css

```diff
@@ -76,10 +76,12 @@ body {
     overflow-y: hidden;
     border: 0;
     border-radius: 20px;
-    width: 400px;
+    /* width: 400px; */
     height: 800px;
     margin: 0 auto;
     text-align: right;
+    background-color: white;
```

Comment:
not really a vital thing but we'll have to delete background-color outta this to accommodate other modals

Reply:
nice catch! deleted the background color and commented the width back in, then pushed to branch again.